### ISSUE TITLE
Perfecting the characters

### ISSUE THREAD

**Issue by larsenwork** [larsenwork/Gidole#25]
I'm revisiting many of the curved characters...any thoughts/feedback?

**Comment by larsenwork** [comment]
![screen shot 2015-03-31 at 15 48 38](https://cloud.githubusercontent.com/assets/3859425/6920225/899fe836-d7bd-11e4-814c-eda0ed4e587a.png)

Some of the updated characters

**Comment by larsenwork** [comment]
new tilde and t
![screen shot 2015-03-31 at 16 49 59](https://cloud.githubusercontent.com/assets/3859425/6921565/ff3f6244-d7c5-11e4-9781-e761545603e2.png)

**Comment by larsenwork** [comment]
Some of the capital letters
![screen shot 2015-03-31 at 18 10 12](https://cloud.githubusercontent.com/assets/3859425/6923348/69ea5bb6-d7d1-11e4-8b1e-25b5e7b19284.png)

**Comment by larsenwork** [comment]
Further tweaking
![screen shot 2015-03-31 at 19 08 52](https://cloud.githubusercontent.com/assets/3859425/6924516/6aadfbea-d7d9-11e4-92c5-15f8bd5dde3b.png)

**Comment by larsenwork** [comment]
Less rounded D, new terminal on J and narrower S
![screen shot 2015-04-01 at 21 16 50](https://cloud.githubusercontent.com/assets/3859425/6949397/732d6874-d8b4-11e4-9ba8-a90a7978ea7e.png)

**Comment by jdittrich** [comment]
do you have some images with comparisons or the like? I’d like to help with feedback, but without knowing the character styles by heart it it is difficult :)

**Comment by larsenwork** [comment]
Thanks for your interest:) I think most of them (if not all) are merged into the default font now but I'll add comparisons to new images I might upload

**Comment by inferno986return** [comment]
Would love comparisons. These glyphs do look lovely, but it's hard to tell if they are an improvement over the original.

**Comment by larsenwork** [comment]
yeah, sorry about that - they were improvements - I didn't take screenshot of the old ones.

hnm: top right part of curved looked a bit thin + not round
t: bottom left part of curve was too thin + lower point was to far to the right (=> looked like the t was falling to the left)
tilde: the curves flow more "effortlessly" now
BDPR: more perfectly rounded and maybe a bit more boxed look
U: the bottom left and right part of the curve was too thin
J: the curved opened up too much before
S: can't remember what I fixed here.... :)

I will post before:after comparisons on future adjustments ;)

**Comment by inferno986return** [comment]
A description is better than nothing. Thanks.

I look forward to seeing more comparisons as this typeface develops.

**Comment by larsenwork** [comment]
##### a more playful l with hook

OLD
![ylu](https://cloud.githubusercontent.com/assets/3859425/6878414/e735fc48-d4dd-11e4-86a7-1665c0fd4850.png)

NEW
![screen shot 2015-04-08 at 00 40 13](https://cloud.githubusercontent.com/assets/3859425/7035366/0022a2f8-dd88-11e4-8b11-ab8267086792.png)

##### Interrobang

V1
![screen shot 2015-04-07 at 21 40 26](https://cloud.githubusercontent.com/assets/3859425/7035375/28232a3e-dd88-11e4-99f6-7b9831737435.png)

V2
![screen shot 2015-04-08 at 00 03 17](https://cloud.githubusercontent.com/assets/3859425/7035379/39021fe0-dd88-11e4-8bb0-6379a6f10980.png)

**Comment by davelab6** [comment]
I compared this to other DIN designs, and I actually am not quite sure why you say this is a DIN :) Almost all the proportions are different, many of the terminals are very different... :)

**Comment by larsenwork** [comment]
@davelab6 Hehe...it started out a lot more "DIN'y" and trying to humanise it. What would you call it today?

**Comment by larsenwork** [comment]
I've thought about "humanist sans serif" but that's just dull...

**Comment by larsenwork** [comment]
note to self: update k and K...not happy with those

**Comment by davelab6** [comment]
I think I would call it a RATIONAL humanist sans :) In the first image you posted:

![](https://cloud.githubusercontent.com/assets/3859425/6920225/899fe836-d7bd-11e4-814c-eda0ed4e587a.png)

The `a` is really humantic sans (frutiger et al) except for that totally level/flat crossbar, and then with all the flat-y sided rounds, which are not totally flat, there's a similar thing going on... so its not 'flat geometric' (as opposed to a round geometric like Avenir/Futura/etc) or a 'flat grotesk' (like just published https://github.com/googlefonts/rubik) 

cc @ebensorkin as he might have fun trying to define this :)

**Comment by davelab6** [comment]
(and that 'rational' personality is core to DIN, so, there's some lineage here, but I don't think it is "a DIN")

**Comment by larsenwork** [comment]
@davelab6 I'm new to all this but the flat-y sided rounds, the (almost) symmetrical arcs in e.g. u, n and m, the wide apertures and smooth curves is what defines this font to me.
I looked at Roboto and SF and felt they were too grotesque and unfriendly and kinda took it from there 
`¯\_(ツ)_/¯`

**Comment by davelab6** [comment]
Yeah, those are what I would call rational grotesks

**Comment by larsenwork** [comment]
@davelab6 yep...the good thing about calling it a "Something DIN" is the SEO/marketing aspect even if some other term might be technically more correct. I asked on Twitter the other day and Marcus' reply kinda confirmed my suspicion: https://twitter.com/larsenwork/status/620854141338931200

**Comment by larsenwork** [comment]
![gidole improvements](https://cloud.githubusercontent.com/assets/3859425/12100813/a053aee6-b331-11e5-8ddb-b9b30eb9659f.jpg)
currently improving some letters, left = old, right = new

**Comment by davelab6** [comment]
k improved with a higher leg. The change in the s is subtle, probably good
:)

Great to see you active again :D

**Comment by larsenwork** [comment]
cheers, and happy new year! I've been crazy active - just not so much with spare time projects :smile:
